### DISCUSSION TITLE
Updating a map should always bring the user to the updated map

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#19327] [Ideas] [answered]
Updating a map can cause it to seemingly disappear as it gets moved from its original position in the beatmap carousel to another location. For example, if you're sorting by difficulty and are updating a 6.8* map that was edited and is now 7.2*, the map will disappear from view. I think it would be better behavior if the game brought the user to the newly updated map.

https://user-images.githubusercontent.com/74088315/180602283-144923e6-3c4a-4528-a352-69469af2b804.mp4

**Comment by peppy** [comment, marked as answer]
Yep, this happens when the beatmap difficulty you have selected changes in hash. It works correctly if the currently selected difficulty is part of an updated set but didn't change itself.

**Reply by frenzibyte** [reply]
Tracking via #19328
